swagger.js

```diff
@@ -32,6 +32,86 @@ const doc = {
         },
       },
     },
+    definitions: {
+      Order: {
+        $_id: 'idabc123456',
+        $user: {
+          $id: 'idabc123456',
+          $name: 'John Doe',
+          $username: 'johndoe',
+          $email: 'john@example.com',
+          $role: { '@enum': ['user', 'admin'] },
+        },
+        $orderStatus: {
+          '@enum': [
+            'pending',
+            'payment_successful',
+            'payment_failed',
+            'cancelled',
+            'complete',
+          ],
+          default: 'pending',
+        },
+        orderItems: [
+          {
+            $album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+            $quantity: 2,
+            $_id: 'idabc12345',
+          },
+        ],
+        $subtotal: 9.99,
+        $tax: 0.75,
+        $total: 10.74,
+        $createdAt: '2023-08-08T20:45:00.942Z',
+        $updatedAt: '2023-08-08T20:45:00.942Z',
+        paymentIntentId: 'pi_abc123456',
+      },
+      NewOrder: {
+        type: 'object',
+        properties: {
+          album: {
+            type: 'string',
+            example: 'Album ID or Name',
+          },
+          quantity: {
+            type: 'integer',
+            example: 2,
+          },
+        },
+      },
+    },
+
+    OrderList: {
+      $id: '789012',
+      orders: [
+        {
+          user: 'Jane Smith',
+          orderItems: [
+            {
+              album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+              quantity: 1,
+            },
+          ],
+          subtotal: 14.99,
+          tax: 1.25,
+          total: 16.24,
+        },
+        {
+          user: 'Bob Johnson',
+          orderItems: [
+            {
+              album: '5f7f1f7a5c3f2b2d9c1b0b9e',
+              quantity: 3,
+            },
+          ],
+          subtotal: 29.97,
+          tax: 2.5,
+          total: 32.47,
```

Comment:
Missing a few properties that are returned in the response:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/36742ba7-a009-4c16-a74a-19a79b339404)

```suggestion
          total: 16.24,
          orderStatus: { '@enum': ['pending', 'payment_successful', 'payment_failed', 'cancelled', 'complete'] },
          createdAt: '2023-08-08T20:45:00.942Z',
          updatedAt: '2023-08-08T20:45:00.942Z',
          paymentIntentId: 'pi_12345abcdef'
        },
        {
          user: 'Bob Johnson',
          orderItems: [
            {
              album: '5f7f1f7a5c3f2b2d9c1b0b9e',
              quantity: 3,
            },
          ],
          subtotal: 29.97,
          tax: 2.5,
          total: 32.47,
          orderStatus: { '@enum': ['pending', 'payment_successful', 'payment_failed', 'cancelled', 'complete'] },
          createdAt: '2023-08-08T20:45:00.942Z',
          updatedAt: '2023-08-08T20:45:00.942Z',
          paymentIntentId: 'pi_12345abcdef'
```